app/javascript/pages/browse/MapBrowser.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <section class="MapBrowser">
+    <mapbox
+        access-token="pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"
```

Comment:
We'll probably want to pull this in via an environment variable.

That'll require reading that on the server and making it available to the front end, which we don't currently have an example of (well, we have examples of getting page data into vue, but i suspect we'll want a slightly different scheme for globals such as this).

I'm ok with punting on that for now. Would you mind leaving a `FIXME` comment in there so we come back to it (one day!).